src/Stryker.CLI/Stryker.CLI/OptionsBuilder.cs

```diff
@@ -35,59 +37,78 @@ public class OptionsBuilder
                         .AddJsonFile(fileLocation)
                         .Build().GetSection("stryker-config");
             }
+
             return new StrykerOptions(
                 basePath,
                 GetOption(reporter.Value(), CLIOptions.Reporters),
                 GetOption(projectUnderTestNameFilter.Value(), CLIOptions.ProjectFileName),
                 GetOption(additionalTimeoutMS.Value(), CLIOptions.AdditionalTimeoutMS),
                 GetOption(excludedMutations.Value(), CLIOptions.ExcludedMutations),
                 GetOption(logLevel.Value(), CLIOptions.LogLevel),
-                GetOption(logToFile.HasValue(), CLIOptions.UseLogLevelFile),
-                GetOption(devMode.HasValue(), CLIOptions.DevMode),
+                GetOption(logToFile.Value(), CLIOptions.LogToFile),
+                GetOption(devMode.Value(), CLIOptions.DevMode),
                 GetOption(maxConcurrentTestRunners.Value(), CLIOptions.MaxConcurrentTestRunners),
                 GetOption(thresholdHigh.Value(), CLIOptions.ThresholdHigh),
                 GetOption(thresholdLow.Value(), CLIOptions.ThresholdLow),
-                GetOption(thresholdBreak.Value(), CLIOptions.ThresholdBreak));
+                GetOption(thresholdBreak.Value(), CLIOptions.ThresholdBreak),
+                GetOption(filesToExclude.Value(), CLIOptions.FilesToExclude));
         }
 
-        private T GetOption<V, T>(V value, CLIOption<T> defaultValue)
+        private T GetOption<V, T>(V cliValue, CLIOption<T> option)
         {
-            if (value != null)
+            if (option.ValueType == CommandOptionType.NoValue && cliValue is string cliNoValueValue && cliNoValueValue == "on")
```

Comment:
Why can't we simply ask for HasValue() on a NoValue type instead of parsing the string value?